suse/rdma-core.spec

```diff
@@ -57,6 +57,7 @@ BuildRequires:  pkgconfig
 BuildRequires:  pkgconfig(libsystemd)
 BuildRequires:  pkgconfig(libudev)
 BuildRequires:  pkgconfig(systemd)
+BuildRequires:  pkgconfig(udev)
 %ifnarch s390 s390x
```

Comment:
Not sure why this is needed.. We don' t need udev installed to build rdma-core, just libudev

Reply:
For SUSE we do. Our buildsystem checks commands in %post.
Without this it fails saying udevadm does not exists